.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: ruby
+rvm:
+  - 2.2.4
+# bundler_args: --without production
+# script:
+#   - bundle exec rake db:test:prepare
+#   - bundle exec rspec spec/
```

Comment:
That's what I was thinking in the back of my mind when you mentioned it, but I wasn't sure. I'll remove all the travis config and just leave the Gemnasium badge.